web/src/main/resources/openmrs-servlet.xml

```diff
@@ -102,7 +102,7 @@
 		<property name="order"><value>98</value></property>
 	</bean>
 
-	<bean class="org.springframework.web.servlet.mvc.SimpleControllerHandlerAdapter"/>
+	<bean class="org.springframework.web.servlet.mvc.SimpleControllerHandlerAdapter" name="simpleControllerHandlerAdapter"/>
```

Comment:
Uh... when I said in my comment the name had to be `org.springframework.web.servlet.mvc.SimpleControllerHandlerAdapter`, I meant it needs to be exactly that string. This just re-creates the issue:

```suggestion
	<bean class="org.springframework.web.servlet.mvc.SimpleControllerHandlerAdapter" name="org.springframework.web.servlet.mvc.SimpleControllerHandlerAdapter"/>
```